app/src/components/new-request/stats.vue

```diff
@@ -111,11 +111,22 @@ export default class Stats extends Vue {
 
   async created (): Promise<void> {
     if (
+    /*
+     * Feature-flag semantics for hardcoded_*_wait_time:
+     * - flag > 0 : use the flag value (explicit positive wait time).
+     * - flag === 0 OR FF disabled: fall back to backend stats.
+     * - flag < 0 : flag indicates the wait time is disabled.
```

Comment:
Using the word "disabled" here again is confusing.

How about saying "flag indicates API value should be used"?